### ISSUE TITLE
Treat preview data as consented

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#499]
**TL;DR**: Treat preview data as automatically consented so that it can be downloaded without going through the consent manager interface.

**Narrative**

"As an Experimenter designing a study, I would like to be able to view preview data to ensure that the correct types of information are being captured or to begin designing an analysis workflow. But I may not have permissions to actually code consent for the study, and preview data in general won't include "valid" consent."

**Acceptance Criteria**

- [ ] Preview data is displayed under Individual Responses and All Responses views, and available via the API to an experimenter with read_preview_data permissions for this study, regardless of whether consent is in Pending, Accepted, or Rejected state. 
- [ ] Non-preview data is not available in the above views or the API if consent is in Pending or Accepted state. 
- [ ] Tests tests tests

**Implementation Notes**

It would also work to still require consent, but allow people with permission to read preview data permission to code preview consent. But that adds another step before taking a quick look at the data, and I'm not sure we actually want to get people in the habit of approving fake consent videos...

**Comment by kimberscott** [comment]
Note - for now we *are* using a separate perm for coding preview consent to keep everything as consistent as possible, but would still like to explore more efficient options.
